AvaTax.TaxModule.Data/Model/AvaCreateTransactionModel.cs

```diff
@@ -49,7 +51,8 @@ public virtual AvaCreateTransactionModel FromContext(TaxEvaluationContext contex
             return this;
         }
 
-        public virtual AvaCreateTransactionModel FromOrder(CustomerOrder order, string requiredCompanyCode)
+        public virtual AvaCreateTransactionModel FromOrder(CustomerOrder order, Store store, string requiredCompanyCode, 
```

Comment:
It is our converter conception violation to pass services into model converters, could you avoid this?